portfolio1.css

```diff
@@ -0,0 +1,142 @@
+body {
+	background-color: rgb(165,42,42);
+	color: white;
+}
+
+a {
+	color: red;
+}
+
+.top-bar {
+	display: flex;
+	justify-content: space-between;
+}
+
+.logo-link {
+	width:15%;
+	height: 85%;
+}
+
+.accessibility-link {
+	background-color: rgb(75,0,130);
+	height: 15%;
+}
+.form {
+	display: flex;
+	flex-direction: column;
+}
+
+button {
+	 color:	rgb(128,0,128);
+	 background-color: rgb(255,0,0);
+	 width: 12%;
+	 font-family: Helvetica, Arial, Times;
+	 font-weight: bold;
+}
+.buttons1 {
+     padding: 1%;
+     display: flex;
+	 justify-content: space-between;
+}
+
+#submit {
+	color: white;
```

Comment:
strange indentations... some are in, others out

Reply:
Thanks for notifying me!

Reply:
i concur